portfolio/src/main/webapp/visualizer.js

```diff
@@ -39,18 +39,19 @@ function drawMap(date) {
   data.addColumn('string', 'State');
   data.addColumn('number', 'Cases');
 
-  document.getElementById("date").textContent = "COVID Cases: " + global_data[i].date;
-  let dailyCasesByState = global_data[i].cases;
+  document.getElementById("date").textContent = "COVID Cases: " + global_covid_data[index].date;
+  let dailyCasesByState = global_covid_data[index].cases;
   Object.keys(dailyCasesByState).forEach((state) => {
       data.addRow([state, dailyCasesByState[state]]);
     });
     const chart = new google.visualization.GeoChart(document.getElementById('regions_div'));
     chart.draw(data, options);
 }
 
+// stores what index in global_data to display
+let currIndexInCases = 0;
 // Draws map of cases for following day
-let i = 0;
-function nextDay() {
-  i++;
-  drawMap(i)
+function changeMapDate(increment) {
+  currIndexInCases += increment;
+  drawMap(currIndexInCases);
```

Comment:
I think a nice UX addition here would be to add the `disabled` attribute to the buttons if it will take them back too far.